api-specificatie/common.yaml

```diff
@@ -453,6 +453,18 @@ components:
               title:
                 type: "string"
                 example: "Volgende pagina"
+          last:
```

Comment:
Ik heb het waarschijnlijk bij een vorige review gemist, maar waarom is er niet voor gekozen om de HALLink te hergebruiken? Nu worden bij code generatie 4 identieke classes (First, Next, Previous en Last) gegenereerd.